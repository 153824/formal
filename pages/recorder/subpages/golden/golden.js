const app = getApp();
Page({
    data: {
        redirect: '',
        goldenText: ''
    },
    onLoad: function (options={redirect: {}}) {
        let redirect = JSON.parse(options.redirect)
        this.setData({
            redirect,
        })
        this.getGolden(redirect.receiveRecordId);
    },
    goToReplying() {
        const {url, evaluationId, receiveRecordId, isChapter} = this.data.redirect;
        const targetURL = isChapter ? `${url}?receiveRecordId=${receiveRecordId}&evaluationId=${evaluationId}` : `${url}?receiveRecordId=${receiveRecordId}`
        wx.redirectTo({
            url: targetURL
        });
    },
    getGolden(receiveRecordId) {
        const that = this;
        app.doAjax({
            url: `../wework/evaluations/${receiveRecordId}/synopsis`,
            method: 'GET',
            success(res) {
                that.setData({
                    goldenText: res.commitment
                })
            },
        })
    }
});
