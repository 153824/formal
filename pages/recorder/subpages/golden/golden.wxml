<hola-title type="cant-back" liner="liner-gray"></hola-title>
<view class="golden" style="margin-top: {{statusbarHeight + titleHeight}}px;">
    <view class="golden-text-wrapper">
        <view class="golden-title">
            <text class="title">作答承诺书</text>
        </view>
        <view class="golden-text-group">
            <text>{{goldenText}}</text>
        </view>
    </view>
    <view class="golden-agree" bind:tap="goToReplying">同意</view>
</view>
