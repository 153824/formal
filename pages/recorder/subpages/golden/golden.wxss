.golden{
    display: flex;
    flex-direction: column;
    height: calc(100vh - 235rpx);
    padding: 60rpx 40rpx;
    background-color: #F7F8FA;
}
.title{
    display: flex;
    flex-direction: column;
    font-size: 44rpx;
    font-family: PingFangSC-Semibold, PingFang SC;
    font-weight: 600;
    color: #2F3038;
    line-height: 60rpx;
}
.golden-title::after{
    content: '';
    display: flex;
    width: 68rpx;
    height: 8rpx;
    background: #353EE8;
    border-radius: 4rpx;
}
.golden-text-wrapper{
    flex: 1;
    padding: 40rpx 40rpx;
    background: #FEFFFE;
    box-shadow: 0px 2px 4px 0px #D8DBEB;
    border-radius: 10px;
}
.golden-text-group{
    display: flex;
    flex-direction: column;
    margin-top: 70rpx;
}
.golden-text-group text {
    font-size: 30rpx;
    font-family: PingFangSC-Regular, PingFang SC;
    font-weight: 400;
    color: #303113;
    line-height: 50rpx;
}
.golden-agree{
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;
    padding: 20rpx 0;
    margin-top: 40rpx;
    font-size: 34rpx;
    font-family: PingFangSC-Semibold, PingFang SC;
    font-weight: 600;
    color: #FFFFFF;
    line-height: 48rpx;
    height: 48rpx;
    background: #353EE8;
    box-shadow: 0rpx 4rpx 8rpx 0rpx rgba(0, 0, 0, 0.3);
    border-radius: 10rpx;
}
