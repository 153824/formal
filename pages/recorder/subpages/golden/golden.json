{
  "usingComponents": {
    "van-button": "/common/vant-weapp/dist/button/index",
    "hola-title": "/common/title/title"
  },
  "navigationStyle": "custom"
}
